### PR TITLE
Add basic UDQ restart test

### PR DESCRIPTION
**Pull request by joakim-hove**
Add a test deck for restart testing of basic model with UDQ

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
jenkins build this please

File: udq_actionx/UDQ_M1_RESTART.DATA [hunk 1]
```diff
@@ -0,0 +1,430 @@
...
+
+SOLUTION
+
+RESTART
+   'UDQ_M1' 5 /
+
+
+RSVD
+ 1500 180.0
+ 4000 180.0  /
```
Comment (bska) [inline udq_actionx/UDQ_M1_RESTART.DATA:244]: This bit should technically not be in a restart case.

Comment (joakim-hove) [reply udq_actionx/UDQ_M1_RESTART.DATA]: OK - fixed

**Comment by joakim-hove** [comment]
jenkins build this please

**Review by bska — APPROVED** [review]
Looks good to me.  I'll merge into master.

**Pull request merged**